docs/ROADMAP.md

```diff
@@ -72,8 +63,8 @@ If you have a feature request or suggestion, please [get in touch](https://githu
 | ------------ | ------ |
 | `arrow`      | Stable |
 | `duckdb`     | Stable |
-| `sqlite`     | Stable |
 | `postgresql` | Stable |
+| `sqlite`     | Beta   |
```

Comment:
```suggestion
| `sqlite`     | Release Candidate |
```

SQLite is already RC. Table formatting will probably need to get spaced for the other rows.